setup.py

```diff
@@ -54,11 +54,11 @@ def readfile(name):
 
 testing_extras = tests_require + ['coverage', 'nose']
 
-base_version = parse_version(VERSION).base_version
+major_version = ".".join(VERSION.split(".")[:2])
```

Comment:
just call it root_version, major already has meaning and base does too according to pkg_resources

Reply:
either that or branch_version.